docs/Makefile

```diff
@@ -20,6 +20,8 @@ DOCKER_RUN_DOCS := docker run --rm -it $(DOCS_MOUNT) -e AWS_S3_BUCKET -e NOCACHE
 # for some docs workarounds (see below in "docs-build" target)
 GITCOMMIT := $(shell git rev-parse --short HEAD 2>/dev/null)
 
+all: docs docs-build docs-draft docs-shell test
```

Comment:
this doesn't really make any sense. - you're making a target that does the same thing again and again, with no useful benefits.